workflows/notes.sh

```diff
@@ -0,0 +1,520 @@
+AZ_RESOURCE_GROUP="dc6-resources"
+CLUSTER_NAME="dc6-amaterasu"
+ARGO_NAMESPACE="argo"
+
+# Set AKS credentials into kubectl config context
+az aks get-credentials --name "${CLUSTER_NAME}" --resource-group "${AZ_RESOURCE_GROUP}" --overwrite-existing
+
+
+argo server --namespaced --managed-namespace argo
+
+argo submit dc6-dev-workflow.yaml --entrypoint train-qdm \
+    -p variable="fakevariable" \
+    -p ref-zarr="scratch/qdm-dev-4gtfv/input/refdf.zarr" \
+    -p hist-zarr="scratch/qdm-dev-4gtfv/input/histdf.zarr" \
+    -p out-zarr="scratch/dev/trained-qdm-model.zarr" \
+    -p n-quantiles=100 \
+    -n argo --watch
+
+
+#   File "/argo/staging/script", line 26, in <module>
+#     QDMdg = sdba.adjustment.QuantileDeltaMapping.from_dataset(qdm_model_ds)
+#   File "/opt/conda/lib/python3.8/site-packages/xclim/sdba/base.py", line 125, in from_dataset
+#     obj = cls.from_string(defstr)
+#   File "/opt/conda/lib/python3.8/site-packages/xclim/sdba/base.py", line 82, in from_string
+#     params[dd["name"]] = cls._parametrizable_classes[
+# KeyError: 'Grouper'
```

Comment:
do we need this bit of commented out code?

Reply:
Ah, no. Thanks for catching that. The entire file should be removed.

(done)